nl_server/routes.py

```diff
@@ -83,29 +57,35 @@ def search_vars():
   queries = request.json.get('queries', [])
   queries = [str(escape(q)) for q in queries]
 
-  default_index_type = current_app.config[config.ENV_KEY].default_indexes[0]
-  idx = str(escape(request.args.get('idx', default_index_type)))
-  if not idx:
-    idx = default_index_type
-
-  registry: Registry = current_app.config[config.REGISTRY_KEY]
-
   skip_topics = False
   if request.args.get('skip_topics'):
     skip_topics = True
 
+  r: Registry = current_app.config[REGISTRY_KEY]
```

Comment:
nit: `r` is easy to miss with github codebrowsing, consider `reg` or `registry`.

Reply:
updated to reg, not using registry to avoid override the module name.